### ISSUE TITLE
Middle-clicking a title opens a second tab with `javascript:void(0)`

### ISSUE THREAD

**Issue by ThiefMaster** [olsh/Feedly-Notifier-Firefox#127]

**Comment by christantoan** [comment]
Same here

**Comment by rotemdan** [comment]
I was just about to report this. It also happens when middle-clicking the 'Feedly Website' link, though only the 'javascript:void(0)' tab opens in this case.

**Comment by gajdipajti** [comment]
The issue exists also with Firefox version 44.0b4. Both Windows and Linux.

**Comment by olsh** [comment]
Can't reproduce.
Have you tried to reproduce the bug in a clean FF profile?

**Comment by ThiefMaster** [comment]
Why not just remove the `href="javascript:void(0)"` from the links and preventDefault the event in the JS event handler that also handles opening the new tab?

**Comment by olsh** [comment]
@ThiefMaster yes, it's the most obvious solution. But preventDefault doesn't work in the popup.

**Comment by malibuzios** [comment]
@olsh 

I'm experiencing this constantly (I have a habit of using the middle click for links, it is very hard to change it) here are some animated screen captures:

![middle click article](https://cloud.githubusercontent.com/assets/17719551/14320450/ca5b7554-fc1d-11e5-8c62-d766597a8bf0.gif)

This also happens when clicking the "Feedly website"
![middle click website link](https://cloud.githubusercontent.com/assets/17719551/14320454/cba8a2ec-fc1d-11e5-8c0f-b2e1d379ecf5.gif)

I'm using `2.10.12` on Firefox 45.0.1 x64, Windows 10 x64. Default theme. All extensions disabled apart from this one.

In any case, I wanted to thank you for making this useful extension, I've become quite dependent on it for several years now..

**Comment by olsh** [comment]
Blocked by [this](https://developer.mozilla.org/en-US/Add-ons/SDK/High-Level_APIs/panel#Panel_limitations).

> Panel limitations
> ...
> Mouse button shortcuts, such as using the middle button to open a link in a new page, don't work.

**Comment by malibuzios** [comment]
@olsh 

I searched but so far I couldn't find a bugzilla issue that tracks this. Somebody did file an issue, though, for [ cmd+click a link in a panel should open the link in a new tab](https://bugzilla.mozilla.org/show_bug.cgi?id=858976) and it is marked as "fixed", though it seems to have the same as middle-click in this extension's panel.
